Lib/test/test_sys.py

```diff
@@ -206,6 +206,18 @@ def test_exit(self):
         self.assertEqual(out, b'')
         self.assertEqual(err, b'')
 
+        # gh-125842: Windows uses 32-bit unsigned integers for exit codes
+        # so a -1 exit code is sometimes interpreted as 4294967295.
+        rc, out, err = assert_python_failure('-c', 'import sys; sys.exit(4294967295)')
+        self.assertIn(rc, (-1, 255, 4294967295))
```

Comment:
Isn't it clearer with hex literals?

```suggestion
        # so a -1 exit code is sometimes interpreted as 0xffff_ffff.
        rc, out, err = assert_python_failure('-c', 'import sys; sys.exit(0xffff_ffff)')
        self.assertIn(rc, (-1, 255, 0xffff_ffff))
```